File: Modul-4/Soal 5/PRAK405-2410817110009-MuhammadAlfiGunawan.py
batas = int(input("Masukkan Batas Baris: "))
lipat = int(input("Masukkan Kelipatan: "))
total = 0

for i in range(1, batas + 1, 1):
    jumlah = 0

    
    for j in range(i, 0, -1): 
        jumlah += j * lipat
        print(f"({j} * {lipat})" , end="")

        if j > 1: print(" + ", end="")

    print(f" = {jumlah}")
    total += jumlah
print(f"{total}")

File: Modul-4/Soal 5/PRAK405-2410817110009-MuhammadAlfiGunawan.c
#include <stdio.h>
int main(){
    int batas, lipat;
    int total = 0, jumlah = 0;

    printf("Masukkan Batas Baris: ");
    scanf("%d", &batas);
    printf("Masukkan Kelipatan: ");
    scanf("%d", &lipat);

    for(int i = 1; i <= batas; i++) {
        jumlah = 0; 

        for(int j = i; j > 0; j--) {
            printf("(%d * %d)", j,lipat); 
            jumlah += j * lipat; 
            
            if (j > 1) {
                printf(" + ");
            }
        }
        printf(" = %d\n", jumlah);
        total += jumlah;
    }
    
    printf("%d", total);
    
}